files_in_directory.py

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------------------
+#
+# Python program to list all files in a directory in Python.
+#
+# AUTHOR  : Anuhya Dasaraju
+# VERSION  DATE             AUTHOR           CHANGES
+# -------  ----             -------          -------
+# 1.0      29th Mar, 2024   Anuhya Dasaraju  Initial version
+#
+# ------------------------------------------------------------------------
+
+import os
+
+def list_files(directory):
+    # Iterate over all files in the directory
+    for filename in os.listdir(directory):
+        # Check if the file is a regular file (not a directory)
+        if os.path.isfile(os.path.join(directory, filename)):
+            print(filename)
```

Comment:
later make this work recursively... Basically dir1/dir2/dir3/filename1 etc cases if we give dir1 it should go inside each dir and list files.